continuationsPlugin/src/test/resources/DefInsideSuspendExpected.txt

```diff
@@ -7,13 +7,10 @@ package continuations {
   () extends Object() { this: continuations.compileFromString$package.type =>
```

Comment:
could we make this file `.scala` for consistency?

Reply:
Sure.

Reply:
done